test-dts/mount.d-test.ts

```diff
@@ -1,6 +1,8 @@
 import { expectError, expectType } from 'tsd'
 import { defineComponent } from 'vue'
 import { mount } from '../src'
+// @ts-ignore
```

Comment:
Is it necessary? TBH I don't like `@ts-ignore` even less in dts tests where we're supposed to check that everything works :) I'd like this to work without it.